packages/hash/frontend/src/pages/[account-slug]/types/new/entity-type.page.tsx

```diff
@@ -0,0 +1,228 @@
+import { Button, TextField } from "@hashintel/hash-design-system/ui";
+import {
+  Box,
+  Container,
+  formHelperTextClasses,
+  inputLabelClasses,
+  outlinedInputClasses,
+  Stack,
+  Typography,
+} from "@mui/material";
+import { useRouter } from "next/router";
+import { FormEvent, useState } from "react";
+import { useBlockProtocolCreateEntityType } from "../../../../components/hooks/blockProtocolFunctions/ontology/useBlockProtocolCreateEntityType";
+import { useLoggedInUser } from "../../../../components/hooks/useUser";
+import { getPlainLayout, NextPageWithLayout } from "../../../../shared/layout";
+import { TopContextBar } from "../../../shared/top-context-bar";
+import { OntologyChip } from "../entity-type/ontology-chip";
+import { PlaceholderIcon } from "../entity-type/placeholder-icon";
+
+const RequiredText = () => (
+  <Box
+    component="span"
+    color={(theme) => theme.palette.blue[70]}
+    display="inline"
+    fontWeight="bold"
+  >
+    Required
+  </Box>
+);
+
+const HELPER_TEXT_WIDTH = 290;
+
+const Page: NextPageWithLayout = () => {
+  const [name, setName] = useState("");
+  const [description, setDescription] = useState("");
```

Comment:
For the sake of consistency, I think we should use `react-hook-form`'s `useForm` instead of handling user inputs manually with states.